Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6232,4 +6232,51 @@ final class BuildPlanTests: XCTestCase {
         let dylibs = Array(buildProduct.dylibs.map({$0.product.name})).sorted()
         XCTAssertEqual(dylibs, ["BarLogging", "FooLogging"])
     }
+
+    func testDefaultVersions() throws {
```

Comment:
Is there a better test than a build plan test to use here @MaxDesiatov?

Reply:
IMO this is good and is reasonably fast as it doesn't require creating or reusing fixtures, which are much slower to run.